javascript/modules/offline/OfflineCreatePackOptions.js

```diff
@@ -33,8 +33,11 @@ class OfflineCreatePackOptions {
     return toJSONString(makeLatLngBounds(ne, sw));
   }
 
-  _makeMetadata () {
-    return JSON.stringify({ name: this.name });
+  _makeMetadata (metadata) {
```

Comment:
I think it would be a good idea to add some quick defensive code here with `delete metadata.name;` to prevent someone from over-writing the always-expected name value being normalized. Other than that, this looks great!

Reply:
You are right, I swapped *name* and *metadata* to prevent *name* from being over-written.

Reply:
Yep that's a better way. So simple, why didn't I see it? :)